main.go

```diff
@@ -306,22 +304,30 @@ func getKeyStore(datastore string, keyName string) *Keystore {
 	return NewKeystore(dirstore, keyName)
 }
 
-func loadTenantToken(datastore string) ([]byte, error) {
+func loadTenantToken(config *menderConfig, datastore string) (raw []byte, err error) {
+
+	if raw = config.GetCustomTenantToken(); len(raw) != 0 {
+		err = nil
+		return
+	}
+
 	dirstore := NewDirStore(datastore)
-	raw, err := dirstore.ReadAll(defaultTenantTokenFile)
+
+	raw, err = dirstore.ReadAll(config.GetDefaultTenantTokenFileName())
```

Comment:
Since we decided to switch to a pure string in `mender.conf`, I don't think we need to load it from any file, which means that `datastore`, `GetDefaultTenantTokenFileName()` and `defaultTenantTokenFileName` can all be removed, as well as this logic.

In fact, since this now becomes much simpler, I think you can just remove the `loadTenantToken` function completely, and just call `config.GetCustomTenantToken()` directly (yey, I love code removal!).